spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultSchedulerService.java

```diff
@@ -166,6 +166,15 @@ public void unschedule(String scheduleName) {
 				this.schedulerServiceProperties.getMaxSchedulesReturned());
 	}
 
+	@Override
+	public ScheduleInfo getSchedule(String scheduleName) {
+		List<ScheduleInfo> result = list().stream()
```

Comment:
Instead of using `list`, can we have a find() method to just fetch the schedule by the given name from the corresponding repository? Having `list` and then iterate over it could be an overkill.

Reply:
Understood.  We don't have a find method available in the SPI.  I can add a story to support this.

Reply:
https://github.com/spring-cloud/spring-cloud-scheduler/issues/17

Reply:
We will need to chat with @sabbyanandan to see when we can work on that story.